cinder/volume/manager.py

```diff
@@ -4924,18 +4996,33 @@ def _connection_create(self,
                 volume.save()
         except exception.CinderException as ex:
             LOG.exception("Model update failed.", resource=volume)
+            trace_utils.trace(
+                ctxt, trace_utils.ACTION_VOLUME_ATTACH,
+                volume, f"Model update failed {str(ex)}", failed=True
+            )
             raise exception.ExportFailure(reason=str(ex))
 
         try:
+            trace_utils.trace(
+                ctxt, trace_utils.ACTION_VOLUME_ATTACH,
+                volume, "call driver initialize_connection"
+            )
             conn_info = self.driver.initialize_connection(volume, connector)
         except exception.ConnectorRejected:
             with excutils.save_and_reraise_exception():
-                LOG.info("The connector was rejected by the volume driver.")
+                trace_utils.trace(
+                    ctxt, trace_utils.ACTION_VOLUME_ATTACH,
+                    volume, "ConnectorRejected.  Volume needs to be migrated"
```

Comment:
I'm not sure I follow what I typo'd here?

Reply:
Double spaces between `ConnectorRejected.` and `Volume needs`

Reply:
Double-space is a way to space sentences in monospace font environments, such as man-pages or code. I think it's a matter of taste/convention/consistency. Just saying, because this reads like a possible misunderstanding.
(@joker-at-work all of @hemna's commit messages do the same.)

Reply:
It's a matter of convention for me.  When I went to grade school back in the 70's, we were taught the proper way to space is double space at the end of a .  It just makes it easier to spot the structure of the written word.   It's a habit I'll never break.